src/subject.ts

```diff
@@ -25,3 +25,42 @@ export function makeSubject<T>(): Subject<T> {
     }
   };
 }
+
+export function makeReplaySubject<T>(numReplays = 1): Subject<T> {
```

Comment:
Interesting that the argument here is `numReplay` not `bufferSize`. This is different to both xstream and RxJS. I'm not opposed to `numReplay`, I just wonder if you designed this explicitly like this. I can understand that cycle/run doesn't need the buffer after the entire pipeline is setup.

Reply:
Because we do not know how many sync requests we need to buffer as that depends on the user, so we would have use some arbitrary value and hope it is big enough.

We know however how many consumers we have: 1, the driver.